### PR TITLE
update hosted_url, add test

### PR DESCRIPTION
**Pull request by ros3bud**
Solves #31. LGTY?

### CODE REVIEW

File: src/claim.coffee [hunk 1]
```diff
@@ -84,7 +84,10 @@ response = (vars, req, res) ->
   event = JSON.parse(res.body)
 
   if res.status == 201 && event?.cert?
-    hosted_url = event.cert.parent_location ? event.cert.location
+    if !event.cert.parent_location? or event.cert.parent_location is ''
```
Comment (alexkwolfe) [inline src/claim.coffee]: I think you can simplify here. How about this?

```coffeescript
hosted_url = event.cert.parent_location or event.cert.location
```

In coffeescript, the `?` operator tests for `null` and `undefined`. The `or` operator tests for "falsey" values. Empty string is a falsey value (as is `null` and `undefined`).

You can try it in the coffee console:

```
coffee> one = ""
''
coffee> two = "two"
'two'
coffee> one ? two
''
coffee> one or two
'two'
coffee> one = null
null
coffee> one ? two
'two'
coffee> one or two
'two'
coffee> one = undefined
undefined
coffee> one ? two
'two'
coffee> one or two
'two'
```

**Comment by ros3bud** [comment]
Ahhh, ok - makes sense. Thanks.

**Comment by ros3bud** [comment]
updated.

**Comment by alexkwolfe** [comment]
LGTM.  

:thought_balloon: Don't forget to rebase.

**Pull request merged**